mmeds/util.py

```diff
@@ -944,13 +946,17 @@ def quote_sql(sql, quote='`', **kwargs):
     return formatted
 
 
-def make_pheniqs_config(reads_forward, reads_reverse, barcodes_forward, barcodes_reverse, mapping_file, o_directory):
+def make_pheniqs_config(reads_forward, reads_reverse, barcodes_forward, barcodes_reverse, mapping_file, o_directory,
```

Comment:
oof. it's on my to do list to do a whole rewrite of this like i did with `strip_error_barcodes`

Reply:
:)